src/main/java/com/spotinst/sdkjava/model/bl/ocean/kubernetes/K8sVirtualNodeGroupDeleteRequest.java

```diff
@@ -0,0 +1,43 @@
+package com.spotinst.sdkjava.model.bl.ocean.kubernetes;
+
+public class K8sVirtualNodeGroupDeleteRequest {
```

Comment:
Is this class needed?

Reply:
Yes, this class is needed.  I will combine all the request classes into one request class instead of three

Reply:
I will rename to use the same Vng naming